src/components/Button/Button.tsx

```diff
@@ -325,6 +329,7 @@ export function Button({
         onKeyPress={onKeyPress}
         onMouseUp={handleMouseUpByBlurring}
         onMouseEnter={onMouseEnter}
+        onMouseLeave={handleMouseLeaveByBlurring}
```

Comment:
This could be overkill but it prevents the focus ring from appearing when a user clicks and holds on a button and drags off of it.

i.e.
![image](https://screenshot.click/screencast_2020-02-14_15-50-37.gif)

Reply:
This feels like overkill

Reply:
Removed 👍